.github/ISSUE_TEMPLATE.md

```diff
@@ -9,3 +25,21 @@
 <!-- Behavior would be without the bug. -->
 
 **Steps to reproduce:**
+  <!-- Give a complete description of how to reproduce the problem. -->
+
+### supporting information:
+
+<!-- The following information MUST be included. -->
+
+ - [ ] `node -v` prints:
+ - [ ] Windows, OS X/macOS, or Linux?:
+
+<!-- For feature requests, uncomment the section below. But first, review the existing feature requests and make sure there isn't one that already describes the feature you'd like to see added: -->
```

Comment:
@optimistanoop  No need to add these as well before giving feature request, the contributor will discuss all these things on the issue and there is no need to rewrite it again here.